docs/resources/access_package_assignment_policy.md

```diff
@@ -76,74 +76,74 @@ resource "azuread_access_package_assignment_policy" "test" {
 ## Argument Reference
 
 - `access_package_id` (Required) The ID of the access package that will contain the policy.
-- `description` (Required) The description of the policy.
-- `display_name` (Required) The display name of the policy.
 - `approval_settings` (Optional) An `approval_settings` block to specify whether approvals are required and how they are obtained, as documented below.
 - `assignment_review_settings` (Optional) An `assignment_review_settings` block, to specify whether assignment review is needed and how it is conducted, as documented below.
-- `can_extend` (Optional) When enabled, users will be able to request extension of their access to this package before their access expires.
```

Comment:
This is the erroneous field, should have been `extension_enabled`. The rest is just ordering.